samples/subsys/usb/hid-keyboard/src/main.c

```diff
@@ -184,11 +184,13 @@ int main(void)
 		return -ENODEV;
 	}
 
+	/* doc device enable start */
```

Comment:
these sort of reminds me of stm32cube generated files, just saying :-)

Reply:
I think [:lines: xxx](https://devopstutodoc.readthedocs.io/en/stable/documentation/doc_generators/sphinx/rest_sphinx/code/literalinclude/literalinclude.html#id12) is hard to maintain.